neo4j/util.go

```diff
@@ -0,0 +1,16 @@
+package neo4j
+
+// ConcatenateStringSlices is a helper method to flatten a [][]string slice into a []sting slice.
+// This can be useful for combining bookmarks of multiple sessions.
+func ConcatenateStringSlices(slices... []string) []string  {
```

Comment:
maybe do not export it, i.e. rename it to `concatenateStringSlices`
we don't want users accidentally call that

Reply:
Don't we? I thought we'd offer them this method that they might need when working with bookmarks of more than 2 sessions.